### PR TITLE
Update to work with newer version of Python and peewee

### PR DESCRIPTION
**Pull request by radusuciu**
Some of these changes are documented here: http://docs.peewee-orm.com/en/latest/peewee/changes.html

### CODE REVIEW

File: landfill.py [hunk 1]
```diff
@@ -285,7 +286,7 @@ def get_pwiz_tables(self, engine, database):
         in the DB.
         '''
         introspector = pwiz.make_introspector(engine, database.database,
-            **database.connect_kwargs)
+            **database.connect_params)
```
Comment (radusuciu) [inline landfill.py:289]: Not sure about this one.

File: landfill.py [hunk 2]
```diff
@@ -273,9 +273,10 @@ def get_tables(self, models):
         '''
         Extract all peewee models from the passed in module
         '''
-        return { obj._meta.db_table : obj for obj in
-                models.__dict__.itervalues() if
-                isinstance(obj, peewee.BaseModel) and
+        return { obj._meta.table_name : obj for obj in
+                models.__dict__.values() if
+                hasattr(obj, '_meta') and
```
Comment (radusuciu) [inline landfill.py:278]: In my app, I was getting an instance of `BaseModel` called `Model` which had no `_.meta`

**Comment by Checksum** [comment]
@radusuciu Thanks, I appreciate the PR. Been a while, but I'll have a look at the changes this weekend.

**Comment by radusuciu** [comment]
Hey you're welcome - neat project!

There are still some issues that I noticed as a I tried to use it, so this PR is a WIP - please don't feel obligated to review.

**Comment by Checksum** [comment]
OK, so this PR is definitely a step in the right direction. I'm merging this so that I can push some more fixes. Will also make an actual migrator script that can be run as a cli. Thanks again @radusuciu!

**Pull request merged**